flink-yarn/src/test/java/org/apache/flink/yarn/YarnResourceManagerTest.java

```diff
@@ -584,6 +590,85 @@ public void testGetCpuExceedMaxInt() throws Exception {
 		}};
 	}
 
+	@Test
+	public void testWorkerSpecContainerResourceAdapter_MatchVcores() {
+		final int minMemMB = 100;
+		final int minVcore = 10;
+		final YarnResourceManager.WorkerSpecContainerResourceAdapter adapter =
+			new YarnResourceManager.WorkerSpecContainerResourceAdapter(
+				getConfigProcessSpecEqualsWorkerSpec(), minMemMB, minVcore, true);
```

Comment:
It is usually easier to understand if one use an enum instead of boolean because one can give the different values expressive names (e.g. `MATCH_VCORES`, `IGNORE_VCORES`).